src/js/stores/MesosStore.js

```diff
@@ -0,0 +1,188 @@
+import {EventEmitter} from "events";
+
+import AppDispatcher from "../AppDispatcher";
+import InfoStore from "../stores/InfoStore";
+import InfoActions from "../actions/InfoActions";
+import InfoEvents from "../events/InfoEvents";
+import MesosActions from "../actions/MesosActions";
+import MesosEvents from "../events/MesosEvents";
+
+const FILES_TTL = 60000;
+const STATE_TTL = 60000;
+const MASTER_ID = "master";
+
+class MesosStore extends EventEmitter {
```

Comment:
We didn't introduced es6-classes yet. Could you write it as the other stores, thanx.